### PR TITLE
Add an article about opening PDF using PDF.js

### PR DESCRIPTION
**Pull request by TsvetinaIvanova**

### CODE REVIEW

**Comment by ggarabedian** [comment]
:+1:

File: how-to/how-to-open-pdf-in-app.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    <td>Code</td>	
+  </tr>
+  <tr>
+	<td>Mobile development type</td>
+	<td>Hybrid (Cordova app)</td>
+  </tr>
+</table>
+
+## Description
+Opening a PDF inside a hybrid app is possible only on iOS devices. For Android and Windows Phone, the device needs to have an external PDF viewer. An alternative approach is to use a JavaScript library, like [PDF.js](https://mozilla.github.io/pdf.js/), to parse and render the PDF file inside the WebView. This article provides basic steps and a sample for opening a PDF file inside your app using PDF.js.
```
Comment (Mike4o) [inline how-to/how-to-open-pdf-in-app.md]: Opening a PDF in a hybrid app is possible out of the box only on iOS devices.

File: how-to/how-to-open-pdf-in-app.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+	```
+7. Use the File plugin to read the PDF file as a `Uint8Array` and pass it to the `PDFJS.getDocument()` function.
+8. Render the parsed PDF content into a `canvas` element on the page.
+
+A working example is available here: [PDF Create/Open sample app](https://github.com/PlatformSupport/pdf-sample).
+
+## Notes
+Steps 7 and 8 describe logic that includes code snippets, which are too long to present in a concise way in this article. You can refer to the [`scripts\open.js`](https://github.com/PlatformSupport/pdf-sample/blob/master/PDF%20Sample/scripts/open.js) file in the [PDF Create/Open sample app](https://github.com/PlatformSupport/pdf-sample).
+
+Note that this example relies heavily on a third-party library. At the time of testing, it worked on all three operating systems supported by Telerik Platform. However, there is no guarantee that this code will continue to work seamlessly with future OS versions. It is recommended to test this functionality in your app with each release of a new OS version and each update of the PDF.js library that you decide to use.
```
Comment (Mike4o) [inline how-to/how-to-open-pdf-in-app.md]: Remove "Note that" -> "This example relies ....

File: how-to/how-to-open-pdf-in-app.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+teampulseid: 
+ticketid: 
+pitsid: 
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code</td>
```
Comment (Mike4o) [inline how-to/how-to-open-pdf-in-app.md]: You can also add -> "Third-party product"

**Comment by TsvetinaIvanova** [comment]
@Mike4o

**Comment by Mike4o** [comment]
:+1:

**Pull request merged**